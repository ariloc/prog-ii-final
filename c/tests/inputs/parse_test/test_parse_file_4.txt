   ESto ES una PRUEba   .   es UNA prueBA.
 nuevamente, ESTO es UNA.   prueba    .prueba
