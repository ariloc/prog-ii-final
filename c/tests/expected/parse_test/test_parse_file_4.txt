esto es una prueba
es una prueba
nuevamente esto es una
prueba
prueba
